docs/spark/spark-queries.md

```diff
@@ -36,6 +36,9 @@ Iceberg uses Apache Spark's DataSourceV2 API for data source and catalog impleme
 | [Snapshots metadata table](#snapshots)           | ✔️        | ✔️          |                                                |
 | [Files metadata table](#files)                   | ✔️        | ✔️          |                                                |
 | [Manifests metadata table](#manifests)           | ✔️        | ✔️          |                                                |
+| [Partitions metadata table](#partitions)         | ✔️        | ✔️          |                                                |
+| [Entries metadata table](#entries)               | ✔️        | ✔️          |                                                |
```

Comment:
Can we remove this line?  Link does not work as we decided not to put in the advanced tables.